pynhm/base/control.py

```diff
@@ -20,6 +26,7 @@ def __init__(
         start_time: np.datetime64,
         end_time: np.datetime64,
         time_step: np.timedelta64,
+        config: dict = None,
```

Comment:
this is where i'm putting the control file contents (it wasnt being saved before). I'm ambivalent/not satisfied with "config" as the attr name for this